velox/type/TimestampConversion.h

```diff
@@ -107,6 +107,15 @@ Status daysSinceEpochFromWeekDate(
     int32_t dayOfWeek,
     int64_t& out);
 
+/// Computes the (signed) number of days since unix epoch (1970-01-01).
+/// Returns UserError status if the date is invalid.
+Status daysSinceEpochFromWeekOfMonthDate(
```

Comment:
Perhaps, extract this function into a separate PR and add a unit test.

Reply:
I assume this comment suggest us opening another PR which contains the implementation and tests for `daysSinceEpochFromWeekOfMonthDate` function.

Reply:
I opened a PR for this. #10604